course_gather/settings/base.py

```diff
@@ -38,6 +38,8 @@
     'django_nose',
     'rest_framework',
     'django_filters',
+    'corsheaders',
+    'debug_toolbar',
```

Comment:
These two (and their respective configurations) should be in dev settings

Reply:
I'll move `debug_toolbar`, but is there any downside to `corsheaders` being enabled in prod?

Reply:
Downside? Not really. It's not being used though, thus imo should be removed.

Reply:
Alright, moved in e13adcc.